src/coreclr/nativeaot/System.Private.CoreLib/src/System/RuntimeType.cs

```diff
@@ -2,11 +2,112 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Reflection;
+using System.Diagnostics.CodeAnalysis;
 
 namespace System
 {
     // Base class for runtime implemented Type
     public abstract class RuntimeType : TypeInfo
     {
+        public override string GetEnumName(object value)
```

Comment:
Make all of these sealed overrides? (RyuJIT will be able to devirtualize if this comes from a typeof or GetType.)